src/types/records.js

```diff
@@ -765,9 +767,9 @@ function readRecordBody(view, header, checkLast) {
     case Constants.RECORD_35:
       return readRecord35(view);
     case Constants.RECORD_37:
-      return readRecord37(view);
+      return isLast() ? {} : readRecord37(view);
```

Comment:
This change may introduce problems in the future.

Right now MINIGR,d94 is the only file available that has these two record types at the end of a list.

Other files out there may have valid records at the end of the list and will be broken by this change.

This change also puts the onus on the rest of the code to check that records 37 and 38 are valid before using them.